src/main/java/duke/gui/MainWindow.java

```diff
@@ -0,0 +1,92 @@
+package duke.gui;
+
+import duke.Duke;
+import duke.exception.DukeException;
+import duke.util.Ui;
+import javafx.fxml.FXML;
```

Comment:
I think there should be an empty line between imports from duke package and JavaFX package to be consistent with the way you have been doing your imports in other files.